python/README.md

```diff
@@ -61,4 +61,13 @@ make some new keys from `os.urandom` bytes.
 
 ### Documentation
 
-For more information see the in-code documentation.
\ No newline at end of file
+For more information see the in-code documentation.
+
+### Test
+
+In the test folder we have scripts that implement test classes based on unittest. To run them, use:
+
+```
+python -m test.my_module_name
+```
+from this directory, where `my_module_name` is the name of the test module to be executed (the name of the file without the .py). 
```

Comment:
If I were to follow these instructions without prior knowledge of the codebase, I would need to dive into the test directory, figure out which files contain tests and then execute them one-by-one from the parent directory. Can we try to simplify this a bit?

For example, since you're suggesting the execution with `-m`, maybe we could have a `__main__.py` in the test directory that runs all tests?

Reply:
This sounds good! Mi only concerns would be whether to have a "clean" `__main__.py` that import classes:

```
from test.glome_test import GlomeTest1, GlomeTest2                                                                                                                                          
if __name__ == '__main__':                                                                                              
    unittest.main() 
```
is a big contradiction with the second point of the (style guide)[https://google.github.io/styleguide/pyguide.html#22-imports]. Nonetheless I would like to do this because:
 - we are already using class import in the `__init__` file(i.e. there is precendent).
 - There is no posibility for module misinterpretation.
 - I see no other way around this problem without altering the very convenient:
```
if __name__ == '__main__':                                                                                              
    unittest.main() 
```

Please let me now what do you think about importing classes.

Reply:
As discussed over chat, I'd prefer the latter, too.